backend/cli/pom.xml

```diff
@@ -81,10 +81,10 @@
                       <goal>wget</goal>
                     </goals>
                     <configuration>
-                      <url>https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.7%2B7/OpenJDK17U-jre_x64_windows_hotspot_17.0.7_7.zip</url>
+                      <url>https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9.1/OpenJDK17U-jre_x64_windows_hotspot_17.0.9_9.zip</url>
                       <unpack>true</unpack>
-                      <outputDirectory>${unpack.dir}/windows</outputDirectory>
-                      <sha256>62e82c1afa7509424813e9cb22d8becfec4346d7505e6d2fc8770af4dcd5b9a0</sha256>
+                      <outputDirectory>${unpack.dir}/windows_x64</outputDirectory>
```

Comment:
The windows JRE was missing in the bundle because of this.

Reply:
As we say in german: _Hoppala_